core/utils/utils.py

```diff
@@ -16,7 +16,7 @@
 def _rdf_base(request):
     host = request.get_host()
```

Comment:
Oh wait, it's in _rdf_base, never mind, probably it's not the source of all the other hosts, haha